src/index.js

```diff
@@ -1,14 +1,14 @@
 import { version } from '../package.json';
-import { toM3u8 } from './toM3u8';
-import { toPlaylists } from './toPlaylists';
-import { inheritAttributes } from './inheritAttributes';
-import { stringToMpdXml } from './stringToMpdXml';
+import { toM3u8 as _toM3u8} from './toM3u8';
```

Comment:
why rename these?

Reply:
So that we can export them with there "real name" lower in the file. `export const toM3u8 = _toM3u8`

Reply:
Theoretically, you should be able to do the following and avoid the prefixes:
```js
export {
  toM3u8,
  inheritAttributes,
  ...
};
```

Reply:
I will test with hls-fetcher to see if it works

Reply:
made the change